job-dsls/jobs/new_downstream_production.groovy

```diff
@@ -120,7 +120,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full downstream production build.*")
+                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) prod downstream.*")
```

Comment:
if we would like to get shorter one then I suggest:
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) full prod build.*")
```
or
```suggestion
                triggerPhrase(".*[j|J]enkins,?.*(execute|run|trigger|start|do) product FDB.*")
```

Reply:
I guess option B but lowercase to be consistent with the others.